contracts/gauges/gauge-adapter/src/msg.rs

```diff
@@ -0,0 +1,105 @@
+use cosmwasm_schema::{cw_serde, QueryResponses};
+use cosmwasm_std::{Addr, CosmosMsg, Decimal, Uint128};
+use cw20::Cw20ReceiveMsg;
+use cw_denom::UncheckedDenom;
+
+#[cw_serde]
+pub struct InstantiateMsg {
+    /// Address that is allowed to return deposits.
+    pub admin: String,
```

Comment:
Maybe let's call this "owner" to differentiate it from the CosmWasm contract admin.

Reply:
Let's update this contract to use `cw-ownable`.